frontend/src/components/Agreements/AgreementMetaAccordion/AgreementMetaAccordion.jsx

```diff
@@ -96,6 +96,10 @@ const AgreementMetaAccordion = ({
                         )}
                         {agreement?.vendor && renderTerm("vendor", "Vendor", agreement?.vendor)}
                     </dl>
+                    <dl className="display-flex flex-justify">
```

Comment:
wrap in feature flag

Reply:
resolved with https://github.com/HHS/OPRE-OPS/pull/3732/commits/e82fbc538efb20c1c5d797c72d5ecea5c470a435